yktrace/src/tir.rs

```diff
@@ -41,10 +41,10 @@ impl<'a, 'm> TirTrace<'a, 'm> {
         // Maps symbol names to their virtual addresses.
         let mut addr_map: HashMap<String, u64> = HashMap::new();
 
-        // To avoid unnecessary stores, inlined function calls store their return value directly
-        // into its final resting place chosen by the caller. To make this happen, we use a stack
-        // of IPlaces to keep track of storage for our return values. The `Call` pushes to the
-        // stack, and the corresponding `Return` pops.
+        // A stack to keep track of where to store return values of inlined calls. When we
+        // encounter `$x = Call(...)` we push `$x` to the stack so that later, when we encounter
+        // the corresponding Return, we can find the correct place to store the return value to (by
```

Comment:
Is the `to` stray?

Reply:
This one?

> the correct place to store the return value to

I almost went with `into`, would that help?

Reply:
I think we can just drop the "to" entirely? It's a bit hard to parse with the the following `(by`.